src/microbe_stage/Microbe.cs

```diff
@@ -906,8 +1009,22 @@ private void ApplyScale(Vector3 scale)
         OrganelleParent.Scale = scale;
     }
 
+    private void ApplyRenderPriority()
+    {
+        var material = Membrane.MaterialToEdit;
+
+        if (material != null)
+        {
+            material.RenderPriority = RenderPriority;
+            Membrane.Dirty = true;
```

Comment:
Why is membrane marked dirty here? Isn't it already using the material? Setting it dirty will rely on the membrane data cache here to not impact performance, as if it isn't cached it would be rebuilt.

Reply:
It does, it is set as the surface material and methods like `Membrane.ApplyHealth` just modify the material directly without reapplying it.

Reply:
I see.